sql/driving_distance/drivingDistance.sql

```diff
@@ -95,7 +95,28 @@ ROWS 1000;
 -- COMMENTS
 
 COMMENT ON FUNCTION pgr_drivingDistance(TEXT, BIGINT, FLOAT, BOOLEAN)
-IS 'pgr_drivingDistance(Single_vertex) -- edges_sql(id,source,target,cost[,reverse_cost]), from_vid, distance [,directed]';
+IS 'pgr_drivingDistance(Single_vertex)
+- Parameters:
+   - Edges SQL with columns: id, source, target, cost [,reverse_cost]
+   - From vertex identifier
+   - Distance from vertex identifier
+- Optional Parameters
+   - directed := true
+- Documentation:
+   - ${PGROUTING_DOC_LINK}/pgr_drivingDistance.html
+';
+
 COMMENT ON FUNCTION pgr_drivingDistance(TEXT, ANYARRAY, FLOAT, BOOLEAN, BOOLEAN)
-IS 'pgr_drivingDistance(Multiple vertices) -- edges_sql(id,source,target,cost[,reverse_cost]), from_vids, distance [,directed, equicost]';
+IS 'pgr_drivingDistance(Multiple vertices)
+- Parameters:
+   - Edges SQL with columns: id, source, target, cost [,reverse_cost]
+   - From ARRAY[vertices identifiers]
+   - Distance from vertices identifiers
+- Optional Parameters
+   - directed := true
+   - equicost := false
+- Documentation:
+   - ${PGROUTING_DOC_LINK}/pgr_drivingDistance.html
+';
+
 
```

Comment:
Is it ok to have empty lines at the end of the document or it does not matter much?  
Some files have trailing empty lines and some do not.

Reply:
Good point @vicennial !

Personally I think one empty last line is a good idea. I'm using `.editorconfig` to make my editor care about this, and there is such a file also in the pgRouting repository: https://github.com/pgRouting/pgrouting/blob/master/.editorconfig#L11

Probably everyone shoud work with an editor supporting this configuration setting.

Reply:
not enforcing that yet.